meetings/2021-06-02.md

```diff
@@ -0,0 +1,34 @@
+
+# Ethereum Open Project PGB Meeting
+## Date:  2 June 2021
+
+**Present:** Dan Burnett (EEA, Chair), Tracy Kuhrt (Accenture), Tas Dienes (EF, Vice Chair), Claudia Rauch (OASIS), Carol Geyer (OASIS), Jan Brezina (EY), Tomasz Stanczak (JSON-RPC Chair), John Wolpert (Baseline TSC Chair), Chaals Nevile (ConsenSys), Jamie Clark (OASIS)
+**Regrets:** Antonio Sabado (Nethermind), Melanie Marsollier (Splunk), Chet Ensign (OASIS), Kyle Thomas (Provide), Dan Kochis (Chainlink), Stefan Schmidt (Unibright)
+
+**Guests:** John Whelan, EEA
+
+* **Agenda Items**
+  * Greet new members 
+  * Agenda review
+  * Past action items
+    * JW: develop TSC governance update document offline in conjunction with PGB members.
```

Comment:
Would be helpful to have noted why (in this case: so we can replace TSC members who disappear between elections)